### ISSUE TITLE
Data lost after Windows 10 sleep

### ISSUE THREAD

**Issue by stevendesu** [notepad-plus-plus/notepad-plus-plus#3293]
### Description of the Issue
When Windows 10 goes to sleep, **sometimes** (still experimenting with repro steps) any unsaved changes to files are lost and cannot be recovered by any means

### Steps to Reproduce the Issue

1. Create a new file or open an existing file
2. Make changes to this file
3. **Leave Notepad++ open** and walk away from the computer and wait for it to go to sleep
4. Come back, wake up the computer, and look at Notepad++

### Expected Behavior

The file should look exactly like it did when I walked away, since Notepad++ hasn't been closed in the mean time

### Actual Behavior

The file has been reverted (or destroyed, in the case of new files). There are no backups in `%AppData%\Notepad++\backup` and the "open recent tabs" feature does not restore it.

For reverted files, ctrl+Z or ctrl+shift+Z fails to restore the file.

### Debug Information

Notepad++ v7.4   (32-bit)
Build time : May 14 2017 - 20:35:30
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 10 (64-bit)
Plugins : mimeTools.dll NppConverter.dll NppExport.dll PluginManager.dll 

This never happened before and started a couple of weeks ago. I update Notepad++ whenever there's a new update available, so I believe it was a recent update that caused it

The issue is severe enough that I've stopped using Notepad++ for the time being. I've lost important client notes, to-do lists, and code snippets as a result of this bug.

**Comment by MetaChuh** [comment]
i was not able to reproduce your issue on notepad++ 7.6.6.

please update your notepad++ to the latest version [<kbd>Notepad++ 7.6.6</kbd>](https://notepad-plus-plus.org/download/v7.6.6.html) and migrate or reinstall your plugins if you update from version 7.5.9 or below.

thank you for your help and best regards.

**Comment by paradoxicalNL** [comment]
I have experienced the above in 2 occasions.
Version: V7.5.6 (64 bit)
OS: Windows 10 Pro
Plugins: XML tools
I'm suspecting this might be related to running under admin privileges, but I was not able to reproduce.

I use sleep mode daily, today I noticed a lot of files are missing from the backup directory, not all files affected however. Starting a certain date until today, all files are missing (10/1/2019 - today). Strange is I can see 1 file that is dated 30/04/2019.

I ran recuva recovery, and found some additional files that I managed to recover.
